plugins/dell-kestrel/fu-dell-kestrel-hid-device.c

```diff
@@ -194,8 +194,12 @@ static gsize
 fu_dell_kestrel_hid_device_get_chunk_size(FuDellKestrelHidDevice *self,
 					  FuDellKestrelEcDevType dev_type)
 {
-	if (fu_dell_kestrel_ec_is_chunk_supported(FU_DEVICE(self), dev_type))
+	if (dev_type == FU_DELL_KESTREL_EC_DEV_TYPE_RMM)
+		return FU_DELL_KESTREL_HID_DEV_NO_CHUNK_SZ;
+
+	if (fu_dell_kestrel_ec_is_chunk_supported(FU_DELL_KESTREL_EC(self), dev_type))
```

Comment:
could we prefix this with `if (dev_type == FU_DELL_KESTREL_EC_DEV_TYPE_PD && fu_dell_kestrel_ec_is_chunk_supported...` perhaps?

Reply:
Some other device types (e.g.: ilan, dpmux, etc.) would visit here, and remain `true` to `any_chunk_sz`.  If we restrict it to `pd` only, they'll be excluded and turned into `no_chunk_sz`.

Reply:
Are they all subtypes of `FuDellKestrelEc`? If so, maybe adding a `FU_IS_DELL_KESTREL_EC(self)` would be a good sanity check?

Reply:
sounds, good.  making it.